doctr/models/artefacts/qrcode.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import cv2
+import numpy as np
+from typing import Tuple, Optional
+
+__all__ = ['QRCodeDetector']
+
+
+class QRCodeDetector:
+
+    """ Implements a QR-code detector.
+    Based on open CV QRCodeDetector. Returns both localization and decoded text.
+    """
+
+    def __init__(
+        self,
+    ) -> None:
+        self.detector = cv2.QRCodeDetector()
```

Comment:
just need to inherit from cv2.QRCodeDetector, so that you don't have to implement the constructor ;)